go/pserver/service.go

```diff
@@ -65,6 +69,46 @@ type Parameter struct {
 	Content     []byte
 }
 
+func float32ToString(b []byte) string {
+	f := make([]float32, len(b)/4)
+	buf := bytes.NewReader(b)
+	err := binary.Read(buf, binary.LittleEndian, &f)
+	if err != nil {
+		return ""
+	}
+	return fmt.Sprintf("%v", f)
+}
+
+func float32ByteToString(c []byte) string {
+	var a []byte
+	var b []byte
+	if len(c) <= 80 {
+		a = c
+	} else {
+		a = c[0:40]
+		b = c[len(c)-40:]
+	}
+
+	var s string
+	s = "top:" + float32ToString(a)
```

Comment:
Maybe don't need `top:` and `tail`, developer probably can understand `[3.3702806e+12 2.142699] ... [3.3702806e+12 2.142699]`, would be better if we can remove `]` and `[`: `[3.3702806e+12 2.142699 ... 3.3702806e+12 2.142699]`

Reply:
Done。